podvm/qcow2/userdata.cfg

```diff
@@ -1,8 +1,15 @@
 #cloud-config
-password: PeerP0d
 ssh_pwauth: true
 chpasswd:
   expire: false
 
 locale: en_US.UTF-8
 locale_configfile: /etc/default/locale
+
+users:
+  - name: peerpod
+    #default password: PeerP0d
+    #openssl passwd -1 PeerP0d
+    passwd: $1$u4tXfju6$lvXwBnkFT4S0depRtcTjo/
```

Comment:
Sorry, with my nominated security focal hat on, I just to want to double check - is this password just a newly generated one that is then locked out, so can't be used to ssh into the peer pod build from this?

Reply:
This is the hash version of plaintext password `PeerP0d` that is used during image creation and post that locked out.
User can change this to anything else but it's a manual process before starting the image build.
I will add an additional comment clarifying the same.